docs/UserGuide.adoc

```diff
@@ -141,10 +141,12 @@ Edits the rating and description address of the 1st place to be `3` and `No desc
 * `edit 2 n/Raffles Hotel t/` +
 Edits the name of the 2nd place to be `Raffles Hotel` and clears all existing tags.
 
+// tag::search[]
 === Searching places by name: `search`
 
 The `search` command helps you to search for places whose names contain any of the given keywords. +
 
+Shortcut: `f` +
```

Comment:
The shortcut for search being the letter `f`, doesn't seem intuitive. Perhaps changing it to `s` would make it more intuitive?

Reply:
`s` is used for select already